db/migrate/20230808142338_create_recipe_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecipeFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipe_foods do |t|
+      t.integer :quantiti
+      t.references :recipes, null: false, foreign_key: true
+      t.references :foods, null: false, foreign_key: true
```

Comment:
Instead of having reference as :foods , please ensure its singular :food, this will ensure proper that your tables in the schema file have an appropriate name on the reference column. ie `food_id` instead of the wrong column name `foods_id`